javascript-constructors/example-constructor.js

```diff
@@ -0,0 +1,12 @@
+function ExampleConstructor() {
+}
+
```

Comment:
per the instructions we should be using the `prototype` property. The `.getPrototypeOf()` method is going to get the prototype of `ExampleConstructor`. That would be the object that `ExampleConstructor` is inheriting from. Instead we want the `prototype` property which will be the object that it's children inherit from. 

eg: `ExampleConstructor.prototype`